Release v3.0.0 (#747)
in-toto/in-toto: CHANGELOG.md

# Changelog

## v3.0.0

This release, most notably, marks stable securesystemslib v1.0.0 as minimum

> requirement. The update requires the previously announced removal of deprecated

Reviewer: ```suggestion
requirement. The update incorporates the previously announced removal of deprecated
```
 
perhaps? "requires" sounds funny

Author: Though wait, I meant that when we update to securesystemslib v1.0.0, we must remove the deprecated args. So the removal is not really incorporated by the dependency update, but necessitated.

> requirement. The update requires the previously announced removal of deprecated
> CLI/API arguments (see below).

Author: What about: 

```suggestion
requirement, and, as a consequence, removes the recently deprecated CLI/API arguments (see below).
```

Reviewer: looks good to me.
